modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc

```diff
@@ -0,0 +1,310 @@
+:page-email: ravi.chillanki@teradata.com
+:description: DBT Feast integration with Teradata
+
+== Overview
+
+This how-to shows an approach to create a data pipeline with Teradata database. While there are several ways a data pipeline can be designed, this document helps to understand how components of two of the popular tools in the software industry, namely link:https://docs.getdbt.com/docs/introduction[DBT] and link:https://docs.feast.dev/[FEAST],
+can be integrated with Teradata Vantage. Teradata's robust 'link:https://docs.teradata.com/r/Enterprise_IntelliFlex_VMware/Teradata-VantageTM-Analytics-Database-Analytic-Functions-17.20[Analytics Database Analytic functions]' are used on the Teradata data source within DBT, for data transformation. The output of which is loaded into FEAST to materialize features that can be used in ML models.
+
+== Introduction
+=== DBT
+link:https://www.getdbt.com/product/what-is-dbt/[dbt] (Data Build Tool) is a data transformation tool that is the cornerstone of the Modern Data Stack. It takes care of the T in ELT (Extract Load Transform). The assumption is that some other process brings raw data into your data warehouse or lake. This data then needs to be transformed.
+
+=== FEAST
+link:https://docs.feast.dev/[Feast] (Feature Store) is a flexible data system that utilizes existing technology to manage and provide machine learning features to real-time models. It allows for customization to meet specific needs. It also allows us to make features consistently available for training and serving, avoid data leakage and decouple ML from data infrastructure.
+
+
+== Prerequisites
+
+* Access to a Teradata Vantage database instance.
+
+* Feast-Teradata library installed locally. See link:https://quickstarts.teradata.com/modelops/using-feast-feature-store-with-teradata-vantage.html#_overview[Feast-Teradata Installation instructions]
+
+* DBT library installed locally. See link:https://docs.getdbt.com/docs/get-started/installation[DBT Installation instructions]
+
+== Objective
+To create a data pipeline with Teradata Vantage as source, perform data transformation on some variables in DBT. The principle transformation of data that we do in DBT is onehotencoding of several columns like gender, marital status, state code etc. On top of that the account type column data will be transformed by performing aggregation operations on the couple of columns. All together generates our desired Analytic_dataset with transformed data. Then we use the transformed dataset output as input in FEAST to store features that can be used to generate training dataset for models.
+
+
+== Getting started
+1. Create a new python environment to manage dbt, feast and their dependencies. Activate the environment:
++
+[source, bash]
+----
+python3 -m venv env
+source env/bin/activate
+----
+
+2. Clone the tutorial repository and cd into the project directory:
++
+[source, bash]
+----
+git clone https://github.td.teradata.com/rc255085/Tdatapipeline.git
```

Comment:
I'm assuming this repo needs to be moved to github.com before we publish this?

Reply:
You are right Adam. Do you mean exactly : 'github.com' ? It is there in 'github.td.teradata.com' like i mentioned